cat/c11-partialsc.cat

```diff
@@ -0,0 +1,110 @@
+C "C++11"
+
+(*
+* Overhauling SC atomics in C11 and OpenCL.
+* M. Batty, A. Donaldson, J. Wickerson. In Proc.
+* 43rd ACM SIGPLAN-SIGACT Symposium on Principles of Programming Languages (POPL), 2016.
+*)
+
+(*
+* This model is based on:
+* https://multicore.doc.ic.ac.uk/overhauling/c11_base.cat
+* https://multicore.doc.ic.ac.uk/overhauling/c11_partialSC.cat
+*)
+
+include "c11_cos.cat"
+include "c11_los.cat"
+
+let asw = IW * (M \ IW)
+let sb = po
+let mo = co & (~NAL * ~NAL)
+
+let cacq = ACQ | (SC & (R | F)) | ACQ_REL | (F & CON)
+let crel = REL | (SC & (W | F)) | ACQ_REL
+let ccon = R & CON
+
+// To be compatible with reads from uninit memory, we use our default definition of fr
+//let fr = rf^-1 ; mo
+include "basic.cat"
+let fr = fr & (_ * ~NAL)
```

Comment:
Now that I see this again, I think it would be better to replace all these lines by `let fr = rf^-1;mo | ([R] \ [range(rf)]);loc;[W]`

Reply:
Updated